index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="main-container">
+      <div class="header">
+        <div class="logo">
```

Comment:
I think it would be to use **a** tag for the logo, to ease navigation for the users

Reply:
Hi, @Samuelwanza!! 👋

- [ ] Kindly use the correct sentence structure. It is hard to understand now. You missed the word `better` in the sentence.

- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.